mipengine/node/monetdb_interface/views.py

```diff
@@ -1,28 +1,19 @@
 from typing import List
-from typing import Union
 
+from mipengine.common.validate_identifier_names import validate_identifier_names
 from mipengine.node.monetdb_interface import common
 from mipengine.node.monetdb_interface.common import connection
 from mipengine.node.monetdb_interface.common import cursor
-from mipengine.node.tasks.data_classes import TableSchema
-from mipengine.utils.validate_identifier_names import validate_identifier_names
 
 
 def get_views_names(context_id: str) -> List[str]:
     return common.get_tables_names("view", context_id)
 
 
-def get_view_schema(view_name: str) -> TableSchema:
-    return common.get_table_schema('view', view_name)
-
-
-def get_view_data(context_id: str) -> List[List[Union[str, int, float, bool]]]:
-    return common.get_table_data("view", context_id)
-
-
 @validate_identifier_names
-def create_view(view_name: str, pathology: str, datasets: List[str], columns: List[str], filters_json: str):
-    dataset_names = ','.join(f"\'{dataset}\'" for dataset in datasets)
+def create_view(view_name: str, pathology: str, datasets: List[str], columns: List[str]):
+    # TODO: Add filters argument
+    dataset_names = ','.join(f"'{dataset}'" for dataset in datasets)
 
     cursor.execute(
         f"CREATE VIEW {view_name} AS SELECT {', '.join(columns)} FROM {pathology}_data WHERE dataset IN ({dataset_names})")
```

Comment:
`columns=', '.join(columns)

 cursor.execute(f"CREATE VIEW {view_name} AS SELECT {columns} FROM {pathology}_data WHERE dataset IN ({dataset_names})")`
 
κρατα τα sql queries όσο πιο καθαρα/ομοιογενή γίνεται